src/core/udb-api.service.spec.js

```diff
@@ -467,7 +467,7 @@ describe('Service: UDB3 Api', function () {
     };
 
     $httpBackend
-      .expectGET(baseUrl + 'organizers/?embed=true&limit=10&q=foo-bar&start=0')
+      .expectGET(baseUrl + 'organizers/?embed=true&limit=10&name=foo-bar&start=0')
```

Comment:
I think this test can just be removed. It's called `should find organizers via advanced query` (advanced query = q), and we already have a `should find organizers by a given name` which tests what you changed here.

Reply:
Can you remove it in a follow-up PR?

Reply:
Yep

Reply:
https://github.com/cultuurnet/udb3-angular/pull/608